docs/snippets/3_examples.snippet

```diff
@@ -6,6 +6,7 @@ Want to jump straight in? Get started with one of our sample applications/templa
 | Doodle Dash       | Real-time sketch-recognition game (see [blog](https://huggingface.co/blog/ml-web-games)) | [link](https://github.com/xenova/doodle-dash) |
 | Code Playground   | In-browser code completion website | [link](./examples/code-completion/) |
 | Semantic Image Search | Search for images with text (Next.js + Supabase) | [link](./examples/semantic-image-search/) |
+| JavaScript        | In-browser image object detection | [link](./examples/javascript/) |
```

Comment:
```suggestion
| Vanilla JavaScript        | In-browser object detection | [link](./examples/vanilla-javascript/) |
```

Perhaps we can rename it to "Vanilla JavaScript", and the thing which is special about this tutorial is that it doesn't use additional frameworks or bundlers.

NOTE: This means we need to rename it elsewhere too.